### PR TITLE
 enable backend search functionality

### PR DESCRIPTION
**Pull request by samwel141**
## Description
Added search feature in backend

**Issue(s) addressed**

- Resolves #

**What kind of change(s) does this PR introduce?**

- [x] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**
No backend search feature
**What is the new behavior?**
Can search wallet trust relationships from the backend using originator, target and initiator wallet name

## Breaking change
No breaking change

### CODE REVIEW

**Comment by dadiorchen** [comment]
@samwel141 I think if you can add an integration test (test under __tests__ folder)  to cover this feature, it would be great, at the end of the day, integration is the last defense line for code quality. 
Do think it is easy to write a int test?

**Comment by samwel141** [comment]
Yes, I have added the integration test for search functionality. check

File: __tests__/trust-relationship-api.spec.js [hunk 1]
```diff
@@ -89,6 +89,31 @@ describe('Trust relationship management', () => {
     expect(res.body.trust_relationships[0]).property('id').to.be.a.uuid('v4'); // Unit test, or use Joi to evaluate entire payload
   });
 
+  it.only('GET /trust_relationships with search parameter', async () => {
```
Comment (dadiorchen) [inline __tests__/trust-relationship-api.spec.js:92]: @samwel141 why you do `only` here? It skips other tests.

**Review by dadiorchen — COMMENTED** [review]
@samwel141 so quick! great job!

**Pull request merged**